app/features/partials/trigger_job_button.erb

```diff
@@ -0,0 +1,11 @@
+<form method="POST" action="/projects_erb/<%= project.id %>/trigger">
+  <% if build %>
+    <input id="sha" type="hidden" name="sha" value="<%= build.sha %>" />
+  <% end %>
+  <input
```

Comment:
This is a bit of a nitpick, but could you please indent these as such [[1]](https://github.com/fastlane/ci/blob/master/app/features/configuration/views/forms/_keys.erb#L6-L13):

```erb
  <input
      class="mdl-button mdl-js-button mdl-button--raised mdl-js-ripple-effect mdl-button--accent"
      type="submit"
      <%= "disabled" if disabled %>
      value="<%= build ? "Re-Run for this commit" : "Trigger Job Now" %>"
  />
```